ingestion/src/metadata/ingestion/source/database/trino/metadata.py

```diff
@@ -177,26 +175,6 @@ class TrinoSource(CommonDbSourceService):
     Trino does not support querying by table type: Getting views is not supported.
     """
 
-    def __init__(self, config, metadata_config):
-        self.trino_connection: TrinoConnection = (
-            config.serviceConnection.__root__.config
-        )
```

Comment:
Wouldn't removing `self.trino_connection` cause an attribute error in `def get_database_names()`?

Reply:
I just updated it, we should use `self.service_connection` since we are doing the same in `CommonDbSourceService.__init__`: `self.service_connection = self.config.serviceConnection.__root__.config`